### DISCUSSION TITLE
It looks like the "Encoding Functions" video in the Review section is cut off. Where is the rest of the video?

### DISCUSSION THREAD

**Discussion by maxgrok** [Cyfrin/security-and-auditing-full-course-s23#223] [Q&A]
I tried looking in the Github, searching on YouTube, and couldn't find the rest of the video. Where is it?

**Comment by EngrPips** [comment]
Hello @maxgrok, You will see a complete video about `Encoding Functions` here https://updraft.cyfrin.io/courses/advanced-foundry/how-to-create-an-NFT-collection/evm-encoding
